xgi/drawing/draw.py

```diff
@@ -5,7 +5,6 @@
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 import numpy as np
-import seaborn as sb  # for cmap "crest"
```

Comment:
I think we need to keep this even if it's "unused", because without it I was getting errors for using the cmap "crest" which is only provided by seaborn (and not matplotlib)